packages/core/src/cli/commands/build.command.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  copyFile, rm, cp, writeFile,
+} from 'fs/promises';
+import { exec } from 'child_process';
+import { rollup, RollupBuild } from 'rollup';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjsRollupPlugin from '@rollup/plugin-commonjs';
+// import typescriptRollupPlugin from '@rollup/plugin-typescript';
+import sucraseRollupPlugin from '@rollup/plugin-sucrase';
+
+import kleur from 'kleur';
+import _ from 'lodash-es';
+import { DmnoCommand } from '../lib/dmno-command';
+
+import { addServiceSelection } from '../lib/selection-helpers';
+import { getCliRunCtx } from '../lib/cli-ctx';
+import { addCacheFlags } from '../lib/cache-helpers';
+import { addWatchMode } from '../lib/watch-mode-helpers';
+import { checkForSchemaErrors } from '../lib/check-errors-helpers';
+import { CliExitError } from '../lib/cli-error';
+import { DmnoBuildInfo } from '../../config-loader/find-services';
+
+const program = new DmnoCommand('build')
+  .summary('build compiled dmno config')
+  .description('Builds DMNO config files into minimal code appropriate for deployed contexts without access to all dependencies')
+  .example('dmno build', 'Build the config using current detected service')
+  .example('dmno build --service service1', 'Build dmno config files for use in service1');
+
+addWatchMode(program); // must be first
+addCacheFlags(program);
+addServiceSelection(program);
+
+program.action(async (opts: {
```

Comment:
most of the logic in here should probably move somewhere else, but just wanted to get it working as a POC.